src/Native/Runtime/unix/PalRedhawkUnix.cpp

```diff
@@ -1174,10 +1174,13 @@ REDHAWK_PALEXPORT uint32_t REDHAWK_PALAPI PalCompatibleWaitAny(UInt32_BOOL alert
     return WaitForSingleObjectEx(pHandles[0], timeout, alertable);
 }
 
+#ifndef _ARM_
```

Comment:
This has always been x86/x64 specific optimization - it may be better to change this to:

```
extern "C" void _mm_pause()  
{
#ifdef defined(_AMD64_) || defined(_X86_)
    __asm__ volatile ("pause");
#endif
}
```